File: 251003/truhan/lab3/publisher/internal/mapper/message.go
package mapper

import (
	messageModel "github.com/Khmelov/Distcomp/251003/truhan/lab3/publisher/internal/model"
)

func MapHTTPMessageToModel(msg messageModel.Message) messageModel.Message {
	return messageModel.Message{
		ID:      msg.ID,
		IssueID: msg.IssueID,
		Content: msg.Content,
	}
}

func MapModelToHTTPMessage(msg messageModel.Message) messageModel.Message {
	return messageModel.Message{
		ID:      msg.ID,
		IssueID: msg.IssueID,
		Content: msg.Content,
	}
}

File: 251001/Shyrynski/Lab3.Core/Models/Message.cs
namespace Lab3.Core.Models
{
    public class Message
    {
        public ulong IssueId { get; } = 0;
        public string Content { get; } = string.Empty;
        private Message(ulong issueId, string content) => (IssueId, Content) = (issueId, content);
        static public Message Construct(ulong issueId, string content)
        {
            return new Message(issueId, content);
        }
    }
}
